schemas/com.snowplowanalytics.monitor/application/jsonschema/1-0-0

```diff
@@ -0,0 +1,112 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for an application entity, attached as context to application status report events",
+  "self": {
+    "vendor": "com.snowplowanalytics.monitor",
+    "name": "application",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+
+  "type": "object",
+  "properties": {
+    "name": {
+      "description": "The name of the application, eg rdb-loader",
+      "type": ["string", "null"],
+      "maxLength": 256
+    },
+    "category": {
+      "description": "The category of the application, eg loader",
+      "type": ["string", "null"],
+      "maxLength": 256
+    },
+    "version": {
+      "description": "The version of the application, eg 1.22.1",
+      "type": ["string", "null"],
+      "maxLength": 32
+    },
+    "instance_id": {
+      "description": "The ID of the application instance; a UUID",
+      "type": "string",
+      "maxLength": 36
+    },
+    "instance_name": {
+      "description": "The name of the application instance, eg com-acme-prod1-rdb-loader-1",
+      "type": ["string", "null"],
+      "maxLength": 256
+    },
+    "inputs": {
+      "description": "The inputs consumed by the application, such as streams etc",
+      "type": ["array", "null"],
+      "items": {
+        "description": "Details about the input",
+        "type": "object",
+        "properties": {
+          "id": {
+            "description": "The ID of the input; a UUID",
+            "type": "string",
+            "maxLength": 36
+          },
+          "type": {
+            "description": "The type of the input, eg stream",
+            "type": "string",
+            "maxLength": 32
+          }
+        },
+        "required": ["id", "type"],
+        "additionalProperties": false
+      }
+    },
+    "outputs": {
+      "description": "The outputs of the application, such as streams, databases, etc",
+      "type": ["array", "null"],
+      "items": {
+        "description": "Details about the output",
+        "type": "object",
+        "properties": {
+          "id": {
+            "description": "The ID of the output; a UUID",
+            "type": "string",
+            "maxLength": 36
+          },
+          "type": {
+            "description": "The type of the output, eg database",
+            "type": "string",
+            "maxLength": 32
+          }
+        },
+        "required": ["id", "type"],
+        "additionalProperties": false
+      }
+    },
+    "cloud": {
+      "description": "The cloud on which the application is running",
+      "type": ["string", "null"],
+      "enum": ["aws", "gcp", "azure", "kafka"]
```

Comment:
Kafka is not a cloud is it?

Reply:
Right. I wanted to make sure the schemas can be used outside of those three clouds as well and Kafka is the most popular option for that. But maybe `private` will be better? Or even `other` -- including any private cloud and on-prem?